controllers/circuitbreakerrules_controller.go

```diff
@@ -53,18 +53,16 @@ const (
 func (r *CircuitBreakerRulesReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	ctx := context.Background()
 	log := r.Logger.WithValues("effectiveNs", r.Namespace, "effectiveCrName", r.EffectiveCrName, "req", req.String())
-	log.Info("receive CircuitBreakerRules")
 
 	if req.Namespace != r.Namespace {
-		log.V(int(logging.WarnLevel)).Info("ignore unmatched namespace.")
+		log.V(int(logging.DebugLevel)).Info("ignore unmatched namespace", "namespace", req.Namespace)
```

Comment:
the "req.Namespace" is already in log fixed prefix.

Reply:
Updated.